vertexproject/synapse
# Update circleci config and remove loop arguments

- Update circleci build configs to run nightly tests and replay tests on python 3.8
- Move nightly 3.7 tests to weekly tests
- Add weekly OSX tests
- Remove extra loop arguments since they are deprecated and should always be the current running loop
- fix axon test error condition
